src/js/stores/ui.js

```diff
@@ -392,8 +415,37 @@ define(function (require, exports, module) {
             if (payload.hasOwnProperty("headerHeight")) {
                 this._headerHeight = payload.headerHeight;
             }
-
-            this._recalculateCenterOffset();
+            
+            if (this._recalculateCenterOffset()) {
+                this._centerCurrentDocumentOnce();
+            }
+        },
+        
+        /**
+         * Center the selected document if panel-resize event occurse. 
+         * 
+         * @private
+         */
+        _handleSelecteDocument: function () {
+            this.waitFor(["application"], function () {
+                this._centerCurrentDocumentOnce();
+            });
+        },
+        
+        /**
+         * Center the selected document once.
+         *
+         * @private
+         */
+        _centerCurrentDocumentOnce: function () {
+            var flux = require("js/main").getController().flux,
```

Comment:
The better way to do this is just `this.flux`. http://fluxxor.com/documentation/stores.html#-store-flux-